index.html

```diff
@@ -151,21 +155,20 @@ <h3 class="partner-title h4 fw-bold mt-4  text-center">Partners</h3>
       <img src="images/samsung.png" alt="chambers">
     </div>
   </section>
-  <footer class="footer d-none d-md-flex  w-full px-4 d-flex flex-md-row py-5">
-    <div class="company d-flex ms-3">
+  <footer class="footer container-md justify-content-center align-items-center d-none d-md-flex
+    flex-md-row py-4 ps-4 mx-auto">
+    <div class="company d-flex ms-5 w-25">
       <img src="images/kenya.png" style="width: 90px; height: 50px;" alt="kenya map">
       <h5 id="companytitle" class="company-title fw-bolder small pt-2">Kenya <br> Creatives Department (KCD)</h5>
     </div>
 
-    <div class="second-column ms-5 ">
+    <div class="second-column ms-5 w-75">
       <p id="secondcolumnp" class="desc fw-bold">2023 Kenya Creatives Department (KCD)</p>
       <p id="secondcolumnlorem" class="w-75 fw-bold">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed nesciunt 
         iste rerum nemo soluta voluptate deserunt? Rerum id itaque maxime!</p>
     </div>
     
   </footer>
```

Comment:
- [ ] Just as the previous reviewer mentioned, kindly center the `footer section` so it follows the layout of the original design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225445483-330f89db-c948-43ac-a8d3-d9675cc7079a.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/225445615-fb0bb990-d9fd-40dc-bbc7-c3b9e97f5090.png)